var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -56,7 +56,7 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
 
     # ACC conflict
     variant("cpu_arch", values=str, default="none", description="Enable CPU Target for ACC")
-    variant("target", values=str, default="none", description="Enable CPU Target for ACC")
+    variant("acc_target", values=str, default="none", description="Enable CPU Target for ACC")
```

Comment:
Thanks for this! I just double checked and in the newest version #41019 we don't have `target` variant anymore.

Reply:
I could try to reproduce the changes there here, but I think it's simpler to leave them as they are. Then we can either rebase #41019 on top of this if this goes in first, or just remove any mention of `babelstream` here if #41019 goes in first.